src/sentry/static/sentry/app/utils/getPreloadedData.ts

```diff
@@ -0,0 +1,12 @@
+export function getPreloadedDataPromise(
+  name: string,
+  slug: string,
+  fallback,
+  isInitialFetch?: boolean
+) {
+  const data = (window as any).__sentry_preload;
+  if (!isInitialFetch || !data || data.orgSlug.toLowerCase() !== slug.toLowerCase()) {
```

Comment:
It might be better to check if `data[name]` is a thenable/Promise. i.e. checking `data[name].then` exists and is a function

Reply:
I should also check that it's thennable, yeah good call 👍